src/lib/orm/entities/MemberEntity.ts

```diff
@@ -0,0 +1,16 @@
+import { BaseEntity, Column, Entity, PrimaryColumn } from 'typeorm';
+import { kBigIntTransformer } from '@utils/util';
+
+@Entity('member', { schema: 'public' })
+export class MemberEntity extends BaseEntity {
+
+	@PrimaryColumn('varchar', { length: 19 })
+	public guildID?: string;
+
+	@PrimaryColumn('varchar', { length: 19 })
+	public userID?: string;
+
+	@Column('bigint', { 'default': () => '0', 'transformer': kBigIntTransformer })
+	public points = 0;
```

Comment:
Since this is a bigint supposedly..shouldn't this (redundant) `= 0` be `= 0n`?

Reply:
It uses bigint, yes, but because `2,147,483,648` (integer's max) isn't enough, `9,007,199,254,740,991` is. Either way if this breaks I can switch it to a bigint, it's just not a need for now, since those amounts are vastly high.

Switching it to a `bigint` type will make the maximum `9,223,372,036,854,775,808`, which is approximately 1000 times `number`'s limit.